%YAML:1.0
---
numDisparities: 80
blockSize: 13
preFilterType: 0
preFilterSize: 9
preFilterCap: 8
minDisparity: 4 
textureThreshold: 13
uniquenessRatio: 0
speckleRange: 17
speckleWindowSize: 26
disp12MaxDiff: 11
lambda: 25
sigma: 2
